src/Renci.SshNet/ScpClient.NET.cs

```diff
@@ -15,6 +16,14 @@ public partial class ScpClient
         private static readonly Regex TimestampRe = new Regex(@"T(?<mtime>\d+) 0 (?<atime>\d+) 0");
 
         /// <summary>
+        /// Transferring UNIX files to Windows may require special treatment.
+        /// Enabling this will remove the following special characters from filenames when copying whole directories:
+        /// " &lt; &gt; | : * ? \ /
+        /// </summary>
+        public bool UnixToWindows { get; set; }
```

Comment:
If we'd be downloading a single file, I'd advise against "fixing" the name and would instruct the user to specify a destination file without these special characters.

In this case,we're recursively downloading a bunch of files, so either we allow an extensible file name mapping mechanism or ensure it "just works".

Why should we only apply it for Windows? I know it's very unlikely to have a character that is valid on some OS but not on unix,, but for consistency we may just as well always apply it.

Note that there's small risk that - by replacing or removing invalid characters - two distinct files on the remote server may end up having the same file name locally.